### PR TITLE
ascii art for bit and byte ordering

### PR DESCRIPTION
**Pull request by paulbastian**

### CODE REVIEW

**Comment by paulbastian** [comment]
after writing this I figured out changing the byte ordering, i.e. starting with byte 0 on the left might be a little more intuitive?

**Comment by tplooker** [comment]
> after writing this I figured out changing the byte ordering, i.e. starting with byte 0 on the left might be a little more intuitive?

Yeah with fresh eyes I definately think that would be better.

**Comment by paulbastian** [comment]
reversed byte order and added the index to the ascii art

File: draft-looker-jwt-cwt-status-list.md [hunk 1]
```diff
@@ -220,9 +247,43 @@ status[9] = 2
...
+           +---+---+---+---+---+---+---+---+  +---+---+---+---+---+---+---+---+  +---+---+...
+  values   | 1 | 1 | 0 | 0 | 1 | 0 | 0 | 1 |  | 0 | 1 | 0 | 0 | 0 | 1 | 0 | 0 |  | 1 | 1 |
+           +---+---+---+---+---+---+---+---+  +---+---+---+---+---+---+---+---+  +---+---+...
+            \     / \     / \     / \     /    \     / \     / \     / \     /    \     /
+             \   /   \   /   \   /   \   /      \   /   \   /   \   /   \   /      \   /
+  status       3       0       2       1          1       0       1       0          3
+
+  index        3       2       1       0          7       6       5       4          11   ...
+
+~~~
```
Comment (c2bo) [inline draft-looker-jwt-cwt-status-list.md:268]:
```suggestion
byte             0                  1                  2
bit       7 6 5 4 3 2 1 0    7 6 5 4 3 2 1 0    7 6 5 4 3 2 1 0
         +-+-+-+-+-+-+-+-+  +-+-+-+-+-+-+-+-+  +-+-+-+-+-+-+-+-+
values   |1|1|0|0|1|0|0|1|  |0|1|0|0|0|1|0|0|  |1|1|1|1|1|0|0|1|
         +-+-+-+-+-+-+-+-+  +-+-+-+-+-+-+-+-+  +-+-+-+-+-+-+-+-+
          \ / \ / \ / \ /    \ / \ / \ / \ /    \ / \ / \ / \ /
status     3   0   2   1      1   0   1   0      3   3   2   1
index      3   2   1   0      7   6   5   4      11  10  9   8
           \___________/      \___________/      \___________/
                0xC9               0x44               0xF9
```

**Review by c2bo — CHANGES_REQUESTED** [review]
More compact version

**Review by c2bo — APPROVED** [review]

**Review by tplooker — APPROVED** [review]

**Pull request merged**